src/animations/heat/main.c

```diff
@@ -1,28 +1,27 @@
 /*
- * 
  * Copyright (C) 2017  Devon Richards
- * ckb-heat is free software: you can redistribute it and/or modify
+ * heat is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 2 of the License, or
  * (at your option) any later version.
  *
- * ckb-heat is distributed in the hope that it will be useful,
+ * heat is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU General Public License for more details.
  * You should have received a copy of the GNU General Public License
- * along with ckb-heat.  If not, see <http://www.gnu.org/licenses/>.
+ * along with heat.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "../ckb/ckb-anim.h"
 #include <math.h>
+#include <ckb-next/animation.h>
 
 void ckb_info(){
     // Plugin info
     CKB_NAME("Heat Map");
     CKB_VERSION("0.2");
     CKB_COPYRIGHT("2017", "RULER501");
-    CKB_LICENSE("GPLv2");
+    CKB_LICENSE("GPLv2+");
```

Comment:
I don't think we're allowed to do this...

Reply:
According to https://github.com/ckb-next/ckb-next/commit/6b8c2b7fc97244b6fee7534953622bd0ae86ecb5 it is provided under GPLv2+, so I just fixed it in the macro. I should do this on master, though.